recipes/ncrystal/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set org = "mctools" %}
+{% set name = "ncrystal" %}
+{% set version = "3.3.0" %}
+{% set sha256 = "0a1e1c818e78bf61a0acf9b941fde8b813d74d1332c399d38198789b5c502219" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  folder: src
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+  skip: true  # [py<36]
+
+requirements:
+  #todo: how do we add dependency on libdl.so? (or we must put -DDISABLE_DYNLOAD=OFF).
+  build:
+    - cmake
+    - make  # [not win]
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+
+test:
+  requires:
+    - python
+    - numpy
+    - cmake
+  imports:
+    - NCrystal
+  commands:
+    - ncrystal-config --help
+    - ncrystal-config -s
+    - nctool --version
+    - nctool --help
+    - nctool --test
+    - ncrystal_inspectfile --help
+    - ncrystal_endf2ncmat --help
+    - ncrystal_hfg2ncmat --help
+    - ncrystal_ncmat2cpp --help
+    - ncrystal_ncmat2hkl --help
+    - ncrystal_onlinedb2ncmat --help
+    - ncrystal_vdos2ncmat --help
+    - ncrystal_verifyatompos --help
+    - nctool --help
+    - ncrystal_inspectfile --help
+    - ncrystal_preparemcstasdir
+    - test -f ./NCrystal_sample.comp
+    - cmake --find-package -DNAME=NCrystal -DCOMPILER_ID=GNU -DLANGUAGE=CXX -DMODE=EXIST -DCMAKE_PREFIX_PATH=$CONDA_PREFIX/lib/cmake
+about:
+  home: https://github.com/{{ org }}/{{ name|lower }}
+  summary: Library for thermal neutron transport in crystals and other materials.
+  description: |
+    NCrystal is a library and toolkit for thermal neutron transport in crystals
+    and other materials. It can be used standalone or as a backend for
+    other Monte Carlo particle simulation frameworks.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: src/LICENSE
+  doc_url: https://github.com/{{ org }}/{{ name|lower }}/wiki
+  dev_url: https://github.com/{{ org }}/{{ name|lower }}
```

Comment:
The use of jinja here obfuscates the recipe and make it hard to follow the links when reading the recipe. Please use the full URLs instead.